tests/constants.py

```diff
@@ -7,3 +7,14 @@
 
 CSV_PATH = os.path.join(TEST_REPO_PATH, INPUT_FOLDER, CSV_ELEMENTS)
 JSON_PATH = os.path.join(TEST_REPO_PATH, INPUT_FOLDER, JSON_FNAME)
+
+# input directory for tests
+TEST_INPUT_DIRECTORY = "inputs"
```

Comment:
you could use `INPUT_FOLDER` below instead of creating a new variable

Reply:
True! But then, if we change the name of the input folder in `src` the tests won't work anymore..